config/application.rb

```diff
@@ -13,11 +13,13 @@
 
 module Panoptes
   class Application < Rails::Application
-    config.autoload_paths += Dir[Rails.root.join('app', 'models', '**/')]
-    config.autoload_paths += Dir[Rails.root.join('app', 'serializers', '**/')]
-    config.autoload_paths += Dir[Rails.root.join('app', 'workers', '**/')]
-    config.autoload_paths += Dir[Rails.root.join('app', 'operations', '**/')]
-    config.autoload_paths += Dir[Rails.root.join('lib', '**/')]
+    config.eager_load_paths += Dir[Rails.root.join('app', 'models', '*/')]
```

Comment:
eager load the code where we can instead of just providing the paths for lookup when searching for constants.

Note: the '*/' to only load the sub dirs!